### ISSUE TITLE
Detection appears to be stuck. Restarting detection process...

### ISSUE THREAD

**Issue by federicotravaini** [blakeblackshear/frigate#1163]
**Describe the bug**
The log reports that the detection is stuck and it keeps restarting.
The detection seems to work fine, but I am not sure why it keeps doing this.

**Version of frigate**
1.13 installed on HASS

**Config file**
Include your full config file wrapped in triple back ticks.
```yaml
mqtt:
  host: 192.168.91.92
  user: user
  password: password

cameras:
  giardino:
    ffmpeg:
      inputs:
        - path: rtmp://192.168.91.100/bcs/channel0_ext.bcs?channel=0&stream=2&user=admin&password=password
          roles:
            - detect
      hwaccel_args:
        - -hwaccel
        - vaapi
        - -hwaccel_device
        - /dev/dri/renderD128
        - -hwaccel_output_format
        - yuv420p
      input_args:
        - -avoid_negative_ts
        - make_zero
        - -fflags
        - nobuffer
        - -flags
        - low_delay
        - -strict
        - experimental
        - -fflags
        - +genpts+discardcorrupt
        - -use_wallclock_as_timestamps
        - '1'
    width: 896
    height: 512
    fps: 5
    objects:
      track:
        - person
    motion:
      mask:
        - 0,512,387,512,385,464,0,464
    snapshots:
      enabled: True
      timestamp: false
      bounding_box: true
      retain:
        default: 7
detectors:
  coral:
    type: edgetpu
```

**Frigate container logs**
```
* Starting nginx nginx
   ...done.
frigate.app                    INFO    : Creating directory: /tmp/cache
frigate.app                    WARNING : Camera giardino has rtmp enabled, but rtmp is not assigned to an input.
Starting migrations
peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
peewee_migrate                 INFO    : There is nothing to migrate
frigate.mqtt                   INFO    : MQTT connected
frigate.app                    INFO    : Camera processor started for giardino: 35
frigate.app                    INFO    : Capture process started for giardino: 36
detector.coral                 INFO    : Starting detection process: 33
frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate.edgetpu                INFO    : TPU found
frigate.watchdog               INFO    : Detection appears to be stuck. Restarting detection process...
root                           INFO    : Waiting for detection process to exit gracefully...
root                           INFO    : Detection process didnt exit. Force killing...
detector.coral                 INFO    : Starting detection process: 66
frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate.edgetpu                INFO    : TPU found
frigate.watchdog               INFO    : Detection appears to be stuck. Restarting detection process...
root                           INFO    : Waiting for detection process to exit gracefully...
root                           INFO    : Detection process didnt exit. Force killing...
detector.coral                 INFO    : Starting detection process: 76
frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate.edgetpu                INFO    : TPU found
frigate.watchdog               INFO    : Detection appears to be stuck. Restarting detection process...
root                           INFO    : Waiting for detection process to exit gracefully...
root                           INFO    : Detection process didnt exit. Force killing...
detector.coral                 INFO    : Starting detection process: 86
frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate.edgetpu                INFO    : TPU found
```

**Computer Hardware**
 - OS: HASS on a Proxmox VM
 - Install method: HASS installer (docker)
 - Virtualization: Proxmox
 - Coral Version: USB
 - Network Setup: Wired

**Comment by blakeblackshear** [comment]
This must mean that frigate is not able to communicate with the Coral. It does appear to detect it, but hangs when trying to perform object detection on it.

**Comment by federicotravaini** [comment]
That's exactly what it looks like (fans keeps running).
What should I try? Any log I can give you? The passthrough seems to work (it is detecting it, so it must be something else)

Any help would be very appreciated!

**Comment by blakeblackshear** [comment]
You could try the coral getting started examples in another VM. If you can get those to work, then frigate should too.

**Comment by federicotravaini** [comment]
I tried to run the started examples in a Ubuntu docker and it didnt work.
I did a logic passthtough of the usb device, but I read that it could cause problems.
I am gonna try to switch to physycal port passthrough and see if something changes.

**Comment by federicotravaini** [comment]
I fixed it.

Here's what I tried:

* as "debug" I tried to run the [Google Coral Sample](https://coral.ai/docs/accelerator/get-started/) on a Ubuntu docker in the same VM where HASS is running. At first, it was nor working.
* I used `lspci` on proxmox host to find the USB ID, and used it in proxmox to passthrough the coral. So i Chose to add a new PCI device, and I did select the USB ID. Reboot the host.
* get back to the ubuntu docker and see if I could run the coral sample. It worked.
* Frigate was working as well

Closing this issue.
